components/user_settings/advanced/join_leave_section/join_leave_section.tsx

```diff
@@ -17,7 +17,7 @@ type Props = {
     currentUserId: string;
     joinLeave?: string;
     onUpdateSection: (section?: string) => void;
-    renderOnOffLabel: (label: string) => string;
+    renderOnOffLabel: (label: string) => string | JSX.Element;
```

Comment:
`React.ReactNode` covers everything renderable (strings, JSX, null, etc), and it ends up being a bit shorter here
```suggestion
    renderOnOffLabel: (label: string) => React.ReactNode;
```

Reply:
Thanks for the suggestion, updated